dozer-api/src/rest/tests/routes.rs

```diff
@@ -53,6 +55,7 @@ async fn count_and_query<S, B, E>(
     path: &str,
     service: &S,
     query: Option<Value>,
+    content_type: Option<ContentType>,
 ) -> (u64, Vec<Value>)
```

Comment:
```suggestion
) -> Result<(u64, Vec<Value>), SomeError>
```

Reply:
Used `StatusCode` as `SomeError`. Asserting that the correct `StatusCode` is received.